app/graphql/ln-update-context.ts

```diff
@@ -0,0 +1,7 @@
+import { createContext, useContext } from "react"
+
+const LnUpdateHash = createContext<string>("")
```

Comment:
I would consider changing the naming to indicate that the LnUpdate is Paid, something like `LnUpdateHashPaid`

Reply:
updated